src/components/Votes.jsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react";
+
+const Votes = ({ newArticle }) => {
+  const [votes, setVotes] = useState(0);
+  const [isClicked, setIsClicked] = useState(false);
+
+  const increase = () => {
+    if (!isClicked) {
+      setVotes(votes + 1);
+      setIsClicked(true);
+    }
+  };
+
+  const decrease = () => {
+    if (!isClicked) {
+      setVotes(votes + 1);
+      setIsClicked(true);
+    }
+  };
+
```

Comment:
You might want to consider putting in functionality so that they are able to _undo_ a mistaken vote! :)